packages/react-components/react-dialog/src/components/DialogContent/useDialogContentStyles.ts

```diff
@@ -14,7 +14,6 @@ export const dialogContentClassNames: SlotClassNames<DialogContentSlots> = {
 const useStyles = makeStyles({
   root: {
     width: '100%',
```

Comment:
in general do we need all these `width: 100%` and `height: 100%`  values across the dialog styles? AFAIK `auto` does the same thing in most cases

Reply:
I'll see to remove both then 👍🏻